gulpfile.js

```diff
@@ -367,6 +367,20 @@ const relayCompiler = gulp.parallel(
       })
       .pipe(gulp.dest(path.join(DIST, 'relay-compiler')));
   },
+  function copyConfigJsonSchema() {
+    return gulp
+      .src(
+        path.join(
+          '.',
+          'compiler',
+          'crates',
+          'relay-config',
+          'src',
+          'config-schema.json',
+        ),
+      )
+      .pipe(gulp.dest(path.join(DIST, 'relay-compiler')));
+  },
```

Comment:
I have never used gulp, but I just copied the approach used for the `schema-extensions.graphql` above. However when I run `yarn gulp release` in the root directory, the output is:
![image](https://user-images.githubusercontent.com/45513122/211009418-5247e072-3c7a-4506-8384-a11d5abac990.png)
I would expect both `schema-extensions.graphql` and `config-schema.json` to be in the root of `relay-compiler`, but they aren't. Am I missing something or is this expected when executing it locally? (I'm on Windows)

Reply:
I'm afraid I'm not sure what's going on here. Maybe you can add some logging to investigate?